rm: Update wireguard configuration (#352)
foundriesio/docs: source/reference-manual/remote-access/wireguard.rst

.. note::

>    Make sure to replace the ``<factory>`` (name of your Factory)

Reviewer: FoundriesFactory

Author: https://github.com/foundriesio/docs/pull/352/commits/80d3f90e5c6fcd06abff5694fa14e06094258535

Configure the Factory with this new service and enable the daemon::

   $ sudo ./factory-wireguard.py \

>        --oauthcreds /root/fiocreds.json \ # where to store the oauth2 credentials

Reviewer: the comment confuses me a bit. I understand that `/root/fiocreds.json` is the path/filename where the oauth2 credentials will be  created. Something like the resultant file with the credentials to be stored.
In case my understanding is right, please ignore my comment, I'm only double checking

Author: https://github.com/foundriesio/docs/pull/352/commits/80d3f90e5c6fcd06abff5694fa14e06094258535

       --factory <factory> \
       --privatekey /root/wgpriv.key \ # where to store generated private key
       enable

> At this point you'll be prompted to authorize an Oauth2 request. It

Reviewer 2: ```suggestion
At this point you will be prompted to authorize an Oauth2 request. It
```

> At this point you will be prompted to authorize an Oauth2 request. It

Reviewer: ```suggestion
At this point you are prompted to authorize an Oauth2 request. It
```

Author: @kprosise - not sure about this suggestion?

Reviewer: at this point you are in the present?

I know we use future a lot in our documentation, but the more verb tenses the more difficult it is to understand

Reviewer 2: I *think* the progressive future tense (will be) is correct. That said, we definitely need to keep things consistent, and I need to brush up on my grammar so I can provide more definitive answers.

Author: you all can decide how you want this and merge at your pleasure.

> At this point you will be prompted to authorize an Oauth2 request. It

Reviewer: ```suggestion
At this point you are prompted to authorize an Oauth2 request. It
```

will look something like:::

   External Endpoint: 165.227.222.126:5555
   VPN Address: 10.42.42.0
   Registring with foundries.io...
   Visit this link to authorize this application:

     https://app.foundries.io/authorize?client_id=fioid_B68H03zaynCXzycBX9M3WKL7xLqJYJyf&response_type=code&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&scope=andy-corp%3Adevices%3Aread-update+andy-corp%3Adevices%3Aread

>     Enter code:  <copy/paste code from app.foundries.io>

Reviewer: Maybe <copy/paste the code from the page in previous link> instead of saying app.foundries.io, as feel like I need to open app.foundries.io and search for a code?

Author: https://github.com/foundriesio/docs/pull/352/commits/80d3f90e5c6fcd06abff5694fa14e06094258535
